### PR TITLE
correct string encoding property

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.config.1.3_fat_tck/publish/tckRunner/tck/pom.xml [hunk 1]
```diff
@@ -123,8 +123,8 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.17</version>
                 <configuration>
+                    <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine> <!-- Needed for ZOS becuase arquillian calls string.getBtytes() -->
```
Comment (hutchig) [inline dev/com.ibm.ws.microprofile.config.1.3_fat_tck/publish/tckRunner/tck/pom.xml]: This line is inside the <configuration> element but outside <systemPropertyVariables> whereas the one above is inside <systemPropertyVariables>

Comment (benjamin-confino) [reply dev/com.ibm.ws.microprofile.config.1.3_fat_tck/publish/tckRunner/tck/pom.xml]: This is the correct one. The one above has been fixed.

**Comment by benjamin-confino** [comment]
Latest build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_RVN-YPfTEeimQc2wZNK2OQ

**Comment by benjamin-confino** [comment]
I am planning to deliver this now and spin off a separate issue for the last remaining bucket.

**Comment by benjamin-confino** [comment]
Zos enabled personal build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_etq_IBTQEemJj5Oykri-vA

File: dev/com.ibm.ws.microprofile.rest.client_fat_tck/publish/tckRunner/tck/tck-suite.xml [hunk 2]
```diff
@@ -12,7 +12,16 @@
     configfailurepolicy="continue">
     <test name="tck-package-org.eclipse.microprofile.rest.client.tck">
         <packages>
+            <method-selectors>
+                <method-selector>
+                    <script language="beanshell">
+                    <![CDATA[
+                        ! (method.getName().contains("testInvokesPostOperation") || method.getName().contains("testInvokesPutOperation"))
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.rest.client_fat_tck/publish/tckRunner/tck/tck-suite.xml]: Please add a comment here to say why these are excluded

**Pull request merged**